app/assets/javascripts/components/CartedProducts.jsx

```diff
@@ -79,7 +91,9 @@ var CartedProducts = React.createClass({
             )}
           </tbody>
         </table>
+        <hr/>
         <div>
+          <p><b>Total: </b>{(this.state.cart_total * 0.01).toLocaleString("en-US", {style: "currency", currency: "USD", minimumFractionDigits: 2})}</p>
```

Comment:
@heydanhey 
Will keep in mind for future PR
Also, it is taking like 2-3 second to render the cart page.. Is this because of the extra database call and method within the controller?